awx/main/tests/functional/api/test_deprecated_credential_assignment.py

```diff
@@ -0,0 +1,308 @@
+import mock
```

Comment:
Decided to write some very explicit tests surrounding now-deprecated credential assignment/retrieval/launch specification.